docs/source/tensors.rst

```diff
@@ -21,6 +21,7 @@ Data type                  dtype                                         CPU ten
 16-bit integer (signed)    ``torch.int16`` or ``torch.short``            :class:`torch.ShortTensor`    :class:`torch.cuda.ShortTensor`
 32-bit integer (signed)    ``torch.int32`` or ``torch.int``              :class:`torch.IntTensor`      :class:`torch.cuda.IntTensor`
 64-bit integer (signed)    ``torch.int64`` or ``torch.long``             :class:`torch.LongTensor`     :class:`torch.cuda.LongTensor`
+Boolean                    ``torch.bool``                                :class:`torch.BoolTensor`     :class:`torch.cuda.BoolTensor`
```

Comment:
nit: can you lower case "Boolean".

Reply:
Looks like the most common appearance is with a capitalized B? (for example, [Wikipedia](https://en.wikipedia.org/wiki/Boolean_data_type) and [Merriam Webster](https://www.merriam-webster.com/dictionary/Boolean))

Reply:
I only suggested that because `integer` and `floating point` are lower cased.

Reply:
Yes, because "integer" and "floating point" are correct spelling when they are lowercased, but "boolean" is generally deemed as incorrect spelling. Imagine we have a table of ["you", "they", "I"], we don't lowercase the "I".

But I can still make the change if it's important to use lowercased 'b' here. Please advise.